guides/common/modules/proc_reset-custom-ssl-certificate-to-default-self-signed-certificate-on-smartproxy.adoc

```diff
@@ -0,0 +1,43 @@
+[id="reset-custom-ssl-certificate-to-default-self-signed-certificate-on-{smart-proxy-context}_{context}"]
+= Reset {customssl} certificate to default self-signed certificate on {SmartProxyServer}
+
+Use this procedure to reset {customssl} certificate to default self-signed certificate on your {SmartProxyServer}.
+
+.On {ProjectServer}
+. Generate a new certificate tar bundle for {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {certs-generate} \
+--foreman-proxy-fqdn _{smartproxy-example-com}_ \
+--certs-tar /root/_{smartproxy-example-com}_-certs.tar.gz \
+--certs-update-all
+----
+. Note down the {foreman-installer} command generated by the above command.
+. Copy the new certificate tar bundle to {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# scp /root/_{smartproxy-example-com}_-certs.tar.gz \
+root@_{smartproxy-example-com}_:/root/_{smartproxy-example-com}_-certs.tar.gz
+----
+
+.On {SmartProxyServer}
+. Ensure you received the certificate tar file from {ProjectServer}.
+. Update the new `katello-ca-consumer-latest.noarch.rpm` package:
```

Comment:
I feel like we need some logic that states if using this RPM. There should be hosts registered via global registration that won't have this and need to use a different method to get the updated certificates.

Reply:
Some more details, `katello-ca-consumer-latest.noarch.rpm` is a deprecated method for configuring a host. There may be existing hosts or new users using this method still; however, there are hosts that are being registered with Global Registration feature. That feature does not use this RPM at all.

Reply:
@ehelms Shall I move with `katello-ca-consumer-latest.noarch.rpm` way for SmartProxy?

Reply:
I think I am saying we need to cover both or make sure the right caveats are present, e.g. `If using katello-ca-consumer....`

Reply:
Reverted from `katello-ca-consumer` to Global Registration for SmartProxy :+1: